cirrus/google_cloud/manager.py

```diff
@@ -153,6 +154,40 @@ def __exit__(self, exception_type, exception_value, traceback):
         self._admin_service = None
         self._storage_client = None
 
+    def open(self):
+        """
+        Run initialization code in __enter__, but do not return self
+        Used for initializing GCM without using `with` block
+
+        Args:
+            -
+        Return:
+            -
+
+        """
+        if self.open_count == 0:
+            self.__enter__()
+        self.open_count += 1
+        return
```

Comment:
Don't need an explicit return if we're not returning anything, Python will return None by default